tests/slip/main.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for ng_tapnet network device driver
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "shell.h"
+#include "shell_commands.h"
+#include "net/ng_netbase.h"
+#include "net/ng_pktdump.h"
+
+/**
+ * @brief   Buffer size used by the shell
+ */
+#define SHELL_BUFSIZE           (64U)
+
+/**
+ * @brief   Read chars from STDIO
+ */
+int shell_read(void)
+{
+    return (int)getchar();
+}
+
+/**
+ * @brief   Write chars to STDIO
+ */
+void shell_put(int c)
+{
+    putchar((char)c);
+}
+
+/**
+ * @brief   Maybe you are a golfer?!
+ */
+int main(void)
+{
+    shell_t shell;
+    ng_netreg_entry_t dump;
+
+    puts("SLIP test");
+
+    /* initialize and register pktdump */
+    dump.pid = ng_pktdump_init();
```

Comment:
I think the syntactical correct call would be to `ng_pktdump_getpid()` here, but `init` works as well, up to you if you want to change this...